### ISSUE TITLE
2049bbs.xyz down?

### ISSUE THREAD

**Issue by thisolddog2017** [2049bbs/2049bbs.github.io#2]
Error 521
Web server is down

**Comment by ghost** [comment]
Yes,because this website's owner will be tried in PRC.(He was jailed in April 19,2020)
Therefore,the police in PRC took control this website and take it down in July 21.
I think this repository may be deleted by PRC government later.
If you‘re safe,please fork this repository to backup it,yours help is important.
